### ISSUE TITLE
[REVIEW]

### ISSUE THREAD

**Issue by vicky157** [EddieHubCommunity/support#3396]
### Resume link

https://drive.google.com/file/d/1CcZBPj0vcsJ06cZqWCH68VcKKY2Gy3Ma/view?usp=sharing

**Comment by emmalearnscode** [comment]
I understand what you've done with the search bar at the top, but I wonder if the sentiment comes across because the rest of the design does not continue in the 'google' style.

You used many colors which is disctracting. Choose just one or two as it makes it easier to know where to look.

Great that you place your internships near the top. I'm sure employers would love to have more details about what you did/learnt at each one.

**Comment by vicky157** [comment]
Yeah sure I will look in to it. Thank you so much for your time Ma'am.
